website/docs/docs/collaborate/data-health-signals.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Data health signals"
+sidebar_label: "Data health signals"
+id: data-health-signals
+description: "Learn how data health signals offer a quick, at-a-glance view of data health when browsing your resources in dbt Explorer."
+image: /img/docs/collaborate/dbt-explorer/data-health-signal.jpg
+---
+
+# Data health signals <Lifecycle status="preview" />
+
+Data health signal offer a quick, at-a-glance view of data health when browsing your resources in dbt Explorer. These icons keep you informed on the status of your resource's health using the indicators **Healthy**, **Caution**, **Degraded**, or **Unknown**.
+
+- Supported resources are [models](/docs/build/models), [sources](/docs/build/sources), and [exposures](/docs/build/exposures).
+- For accurate health data, ensure the resource is up-to-date and had a recent job run.
+- Each data health signal reflects key data health components, such as test success status, missing resource descriptions, missing tests, absence of builds in 30-day windows, [and more](#data-health-signal-criteria)
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-health-signal.jpg" width="55%" title="View data health signals for your models."/> 
+
+## Access data health signals
+
+Access data health signals in the following places:
+- [Search function](/docs/collaborate/explore-projects#search-resources) or click on **Models**, **Sources** or **Exposures** under the **Resource** tab. 
+   - For sources, the data health signal also indicates the source freshness status.
+- View the icons under the **Health** column in [each resource's details page](#view-resource-details). Hover over or click the data health signal to view detailed information.
+- Public models tables under the **Health** column.
+- In the DAG lineage graph, click any node to open the node details panel where you can view the health signal and its details.
```

Comment:
nit: This point is inconsistent with the formatting of the one above ("[Location]" vs. "In the [location]"). I'm not sure which is preferred but I feel like they should follow the same format